css/qmwp-rating-faces.css

```diff
@@ -50,12 +50,12 @@
     -webkit-animation-fill-mode: forwards;
 }
 
-#sectionRateMood ul {
+#sectionRatingFaces ul {
     margin: 0;
     padding: 0;
 }
 
-#sectionRateMood li {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [float can't be used with display: inline-block.](https://www.codacy.com/app/m_3/QuantiModo-WordPress-Plugin/file/1045701984/issues/source?bid=2836188&fileBranchId=2882408#l60)